components/cards/Post.js

```diff
@@ -65,67 +72,93 @@ export default function Post({ post }) {
-      <View style={styles.postActions}>
-        <Button
-          mode="text"
-          onPress={handleLike}
-          color={liked ? theme.colors.accent : theme.colors.text}
-        >
-          {`${liked ? 'Drop' : 'Lift'} ${likeCount}`}
-        </Button>
-        <Button
-          mode="text"
-          onPress={() => {
-            navigation.navigate('Comment', {
-              post,
-              comments: post.comments,
+    <Card style={styles.postContainer}>
+      <Card.Content>
+        <View style={styles.postHeader}>
+          <TouchableOpacity onPress={() => {
+            navigation.navigate('Profile', {
+              userId: post.user.user_id.toString(),
             });
           }}
-        >
-          {`Comment ${post.comments.length}`}
-        </Button>
-
-        <Button mode="text" onPress={() => {}}>
-          Share
-        </Button>
-      </View>
-    </Surface>
+          >
+            <Image
+              style={styles.profilePhoto}
+              source={{ uri: post.user.profilePhoto }}
+            />
+          </TouchableOpacity>
+          <TouchableOpacity onPress={() => {
+            navigation.navigate('Profile', {
+              userId: post.user.user_id.toString(),
+            });
+          }}
+          >
+            <Text style={styles.username}>{post.user.username}</Text>
+          </TouchableOpacity>
+          <TouchableOpacity onPress={() => {
+            navigation.navigate('Community', {
+              community: post.community,
+            });
+          }}
+          >
+            <Text style={styles.community}>{post.community.name}</Text>
+          </TouchableOpacity>
+          <View style={styles.dateContainer}>
+            <Text style={styles.date}>
+              {formatDistanceToNow(new Date(post.date), { addSuffix: true })}
+            </Text>
+          </View>
+        </View>
+        <Text style={styles.postContent}>{post.content}</Text>
+        <View style={styles.postActions}>
+          <View style={{ flexDirection: 'row', alignItems: 'center', marginRight: 15 }}>
+            <TouchableOpacity onPress={handleLike}>
+              <Image
+                source={liked ? dropIcon : liftIcon}
+                style={{
+                  width: 24,
+                  height: 24,
+                  tintColor: liked ? theme.colors.accent : theme.colors.text,
+                  resizeMode: 'contain',
+                }}
+              />
+            </TouchableOpacity>
+            <Text style={{ marginLeft: 4 }}>{likeCount}</Text>
+          </View>
+          <View style={{ flexDirection: 'row', alignItems: 'center' }}>
+            <Pressable
+              onPress={() => {
+                navigation.navigate('Comment', {
+                  post,
+                  comments: post.comments,
+                });
+              }}
+              style={({ pressed }) => ({ opacity: pressed ? 0.6 : 1 })}
+            >
```

Comment:
can this also be TouchableOpacity? or does it have to be different than the rest?